### DISCUSSION TITLE
Celebrate the little things; big web perf win for lazy Yari images

### DISCUSSION THREAD

**Discussion by peterbe** [mdn/mdn-community#494] [Content]
tl;dr:

- **All the redirects are gone!**
- **91KB instead of 262KB of less transferred binary image payload**
- **Now all the live samples depend on a "local data" making it perfectly possible to render this page offline**
- **Bonus; Yari actually CI checks every single image so corrupt or orphaned images can not exist anymore.**

Open https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors with the Network panel and you'll see:
<img width="1503" alt="Screen Shot 2021-06-15 at 10 27 46 AM" src="https://user-images.githubusercontent.com/26739/122071013-66634d00-cdc4-11eb-9488-88851823f1de.png">

Soon, all the live samples' images will be local to the content. I.e. instead of `https://mdn.mozillademos.org/files/1234/Canvas-rgba.png` it's now just `./canvas-rgba.png`. 
It's being done in https://github.com/mdn/content/pull/6025
But you can test it on: https://pr6025.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors

Instead, this is what you see:
<img width="1502" alt="Screen Shot 2021-06-15 at 10 30 22 AM" src="https://user-images.githubusercontent.com/26739/122071526-c954e400-cdc4-11eb-82e1-751263a2a51d.png">

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
